src/Microsoft.VisualBasic.Core/ref/Microsoft.VisualBasic.Core.cs

```diff
@@ -100,6 +100,17 @@ public sealed partial class DateAndTime
         public static System.DateTime Now { get { throw null; } }
         public static System.DateTime Today { get { throw null; } }
     }
+    public sealed partial class ErrObject
+    {
+        internal ErrObject() { }
+        public int Erl { get { throw null; } }
+        public int LastDllError { get { throw null; } }
```

Comment:
We try to keep these files close to how genapis.exe creates them, ie alphabetically ordered members, since sometimes we recreate them. Optional...